src/components/layout/layout.tsx

```diff
@@ -28,6 +28,8 @@ const SCROLL_VIEW_PROPS: ScrollIntoViewOptions = {
 	block: "center",
 };
 
+const isDebug = process.env.NEXT_PUBLIC_ML_DEBUG;
```

Comment:
I suggest to avoid dotenv. We can accomplish analytics control (or any kind of conditional runtime config) via command line arguments or the actual env on the machine (e.g. we can run the client in no-analytics mode).

Reply:
This is not dotenv (which is currently included in package.json for testing purposes), rather, this is native NextJS technique for env vars.

Reply:
On second thought, Did you mean `dotenv` the lib, or `.env` as in `process.env`?
Anyway, my previous comment stands - this is natively supported by NextJS.

Reply:
From the docs: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser